scripts/prefab_descriptors/inspectacleshat.lua

```diff
@@ -0,0 +1,27 @@
+local function Describe(self, context)
+	local player = context.player
+	if not player or
+		player.prefab ~= "winona" or
+		not player.components.skilltreeupdater or
+		not player.components.skilltreeupdater:IsActivated("winona_wagstaff_1") or
+		not player.components.inspectaclesparticipant
+	then
+		return nil
+	end
+
+	if not player.components.inspectaclesparticipant.cooldowntask then
+		return {
+			priority = 0,
+			name = "inspectacleshat",
+			description = context.lstr.inspectacleshat.ready_to_use,
+		}
+	else
```

Comment:
This entire return logic here can and should be condensed. No reason to separate on a condition like that.